flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/flight_model.cfg

```diff
@@ -218,7 +218,6 @@ aileron_to_rudder_scale = 0 ; non-dimensional
 flap_to_aileron_scale = 0 ; non-dimensional
 fly_by_wire = 0 ; Fly-by-wire available true/false
 elevator_elasticity_table = 0:1, 90:1, 200:0.5, 400:0.33
-aileron_elasticity_table = 0:1, 90:1, 200:0.5, 400:0.33
```

Comment:
Guess you found you could remove this table entirely without ill effects? I only ask this because the SDK says it is required (for some unknown reason). You might want to retain it in the flight_model.cfg file for completeness; just change it to 0:1, 400:1, or maybe comment it out.

Reply:
Yes, in the debug menu I found that it will replace it with something like 0:1, 100:1 (the speed values could be different, but you get the idea)